tests/phpunit/DefaultManifestGeneratorTest.php

```diff
@@ -0,0 +1,183 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractor\Tests;
+
+use Keboola\DbExtractor\Adapter\Metadata\MetadataProvider;
+use Keboola\DbExtractor\Adapter\ValueObject\ExportResult;
+use Keboola\DbExtractor\Adapter\ValueObject\QueryMetadata;
+use Keboola\DbExtractor\Manifest\DefaultManifestGenerator;
+use Keboola\DbExtractor\Manifest\ManifestGenerator;
+use Keboola\DbExtractor\TableResultFormat\Metadata\Builder\ColumnBuilder;
+use Keboola\DbExtractor\TableResultFormat\Metadata\Builder\TableBuilder;
+use Keboola\DbExtractor\TableResultFormat\Metadata\Manifest\DefaultManifestSerializer;
+use Keboola\DbExtractor\TableResultFormat\Metadata\ValueObject\ColumnCollection;
+use Keboola\DbExtractorConfig\Configuration\ValueObject\ExportConfig;
+use Keboola\DbExtractorConfig\Configuration\ValueObject\InputTable;
+use PHPUnit\Framework\Assert;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+class DefaultManifestGeneratorTest extends TestCase
```

Comment:
Tym ze som generovanie manifestu oddelil do `DefaultManifestGenerator` tak to mozem uz aj otestovat.